tests/test-dirs/type-enclosing/jane-street.t/run.t

```diff
@@ -54,7 +54,7 @@ escape characters in string literals, so we use the revert-newlines script.
   type t3 : value = D | E of int
       ^
   With verbosity 0: "type t3 = D | E of int"
-  With verbosity 1: "type t3 = D | E of int"
+  With verbosity 1: "type t3 : value = D | E of int"
```

Comment:
In general, I find the diff in this file to be disappointing. The printing of layouts was specifically designed to not print `: value` unless the user wrote it. Do you know why it's now being printed spuriously?

Reply:
I've reverted this behavior

Reply:
Thanks.